Lobby/EOSLobby.cs

```diff
@@ -0,0 +1,197 @@
+﻿using EpicTransport;
+using Epic.OnlineServices.Lobby;
+using UnityEngine;
+using Epic.OnlineServices;
+using System.Collections.Generic;
+
+public class EOSLobby : MonoBehaviour
+{
+    [HideInInspector] public bool ConnectedToLobby = false;
+    public LobbyDetails ConnectedLobbyDetails = new LobbyDetails();
+    
+    [SerializeField] public string[] AttributeKeys = new string[0];
```

Comment:
Why do we serialize the field? T show it in the inspector?
Maybe instead of setting those in the inspector we could provide public methods to add attributes?

Reply:
I did that for visualization purposes. It would be easier to take a look at the inspector for what keys are available rather than trying to memorize what you added via a method.